src/pages/index/atoms/page.ts

```diff
@@ -0,0 +1,14 @@
+import { atom } from "jotai";
+import { searchAtom } from "./setting";
+
+export const sidePageStatusAtom = atom<boolean>(false);
```

Comment:
I don't know what the meaning of `true` and `false` here

Reply:
hmm.. It means that the side page is opened

Reply:
In this case my preference is to use a string union. (e.g. `'visible' | 'hidden'`)
I think it would be okay if you just add the comment.

Reply:
✅ [`8a90b37` (#20)](https://github.com/pbkit/pbkit-devtools/pull/20/commits/8a90b37346e1e5494866752d4299538adf56131e)